mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -9,6 +9,16 @@
 class CrossEntropyLoss(BaseWeightedLoss):
     """Cross Entropy Loss.
 
+    Support two kinds of labels and their corresponding loss type. It's worth
+    mentioning that loss type will be detected by the shape of ``cls_score``
+    and ``label``.
+    1) Sparse label: integers in range [0, num_classes - 1].
+    2) Non-sparse label(one-hot like labels): floats in range [0, 1]. It shares
```

Comment:
What's the difference between soft-label and non-sparse label?

Reply:
I couldn't tell the difference between non-sparse label/soft-label/one-hot like label. I just don't know which term is more former and meaningful.

Reply:
let's use soft-label